axum-core/src/macros.rs

```diff
@@ -49,12 +49,15 @@ macro_rules! __define_rejection {
 
         impl $crate::response::IntoResponse for $name {
             fn into_response(self) -> $crate::response::Response {
+                let status = self.status();
+                let body_text = self.body_text();
+
                 $crate::__log_rejection!(
                     rejection_type = $name,
-                    body_text = $body,
-                    status = http::StatusCode::$status,
+                    body_text = body_text,
+                    status = status,
```

Comment:
I think these macro args are just forwarded to tracing, right? In that case, we can replace `x = x` with just `x` for args.

Reply:
yes and no. the macro-by-example pattern still expects them with the `x = x` syntax unfortunately, and it didn't feel worth it to add a dedicated pattern case for the shorthand syntax.